Sources/App/Views/PackageController/PackageShow+View.swift

```diff
@@ -182,8 +184,42 @@ enum PackageShow {
                         source: SiteURL.package(.value(model.repositoryOwner),
                                                 .value(model.repositoryName),
                                                 .readme).relativeURL(),
-                        // Until the content is loaded, substitute a spinner.
-                        .spinner()
+                        .group(
```

Comment:
I don't think this `group` is needed.

Reply:
There's the very briefest moment when both tab pages are loaded. I think we should add the `hidden` class here at the server so both are hidden, and then one is shown.